src/migrate_openacc_2_openmp_convert.py

```diff
@@ -1194,14 +1193,14 @@ def translate_oacc_2_omp_acc_end_host_data(txConfig, c, carryOnStatus):
 	elif txConfig.HostDataBehavior == CONSTANTS.HostDataBehavior.TARGET_UPDATE:
 		omp_construct = ["target update"]
 		if "host_data" not in carryOnStatus:
-			print (f"Error! Cannot find the matching opening construct for '{construct}'")
+			print (f"Error! Cannot find the matching opening construct for '{omp_construct}'")
 			sys.exit (-1)
 		# Grab the variables used in the opening section
 		variables = carryOnStatus["host_data"]
 		if len(variables) > 0:
 			omp_clauses.append (f"to({variables})")
 		else:
-			print (f"Error! The matching opening construct for '{c.construct}' does not include variables.")
+			print (f"Error! The matching opening construct for '{omp_construct}' does not include variables.")
```

Comment:
Same as above. This must be c.construct rather than omp_construct. I want to point to the original OpenACC code.